CMakeLists.txt

```diff
@@ -73,6 +73,7 @@ option(BUILD_PROFILES "Build the files for profiling" OFF)
 option(BUILD_TESTS "Build the unit tests" OFF)
 option(BUILD_DOC "Build documentation" OFF)
 option(USER "Switch installation folder" OFF)
+option(BUILD_SANDBOX "If on, builds the sandbox" OFF)
```

Comment:
That's nicer than the previous solution! Why do we need the sandbox to be part of the repository though? I.e. what does this bring that could not be done with this?

```bash
vim example.cpp
g++ example.cpp -Ilibrascal/src -Ilibrascal/build/external/Eigen3 -Llibrascal/build -lrascal
```

Reply:
I like to avoid starting from scratch every time I develop something. Also I would prefer to stay within the cmake environment so that debuging this file is the same as other files of the library. This folder is only part of the early development phase and will be removed at some point.